### PR TITLE
Add Linea Sepolia Testnet

### PR DESCRIPTION
**Pull request by k4ctuss**
### Description

Add Linea sepolia chain
Checked the doc for the right token, logo (and minimified), the right
explorer/api and type.
One changeset to minor, ran prettier.

### Backward compatibility

No

### Testing
yes

### CODE REVIEW

File: chains/lineasepolia/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api-sepolia.lineascan.build/api
+    family: etherscan
+    name: Linea Sepolia Testnet Explorer
+    url: https://sepolia.lineascan.build/
```
Comment (paulbalaji) [inline chains/lineasepolia/metadata.yaml:6]:
```suggestion
    url: https://sepolia.lineascan.build
```

File: chains/lineasepolia/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+chainId: 59141
+deployer:
+  name: k4ctuss
+  url: https://github.com/k4ctuss
+displayName: Lineasepolia
```
Comment (paulbalaji) [inline chains/lineasepolia/metadata.yaml:11]:
```suggestion
displayName: Linea Sepolia
```

File: chains/lineasepolia/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api-sepolia.lineascan.build/api
+    family: etherscan
+    name: Linea Sepolia Testnet Explorer
```
Comment (paulbalaji) [inline chains/lineasepolia/metadata.yaml:5]:
```suggestion
    name: Linea Sepolia Explorer
```

**Review by xeno097 — DISMISSED** [review]

File: .changeset/slimy-poets-walk.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': minor
+---
+
+add Linea Sepolia Testnet
```
[the remark this answers is not in the document]
Comment (k4ctuss) [reply .changeset/slimy-poets-walk.md]: Done thanks !

File: chains/gnosischiadotestnet/metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://gnosis-chiado.blockscout.com/api/v2
+    family: blockscout
+    name: Chiado explorer
+    url: https://blockscout.chiadochain.net
```
[the remark this answers is not in the document]
Comment (k4ctuss) [reply chains/gnosischiadotestnet/metadata.yaml]: you're right thanks again

File: chains/gnosischiadotestnet/metadata.yaml [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://gnosis-chiado.blockscout.com/api/v2
+    family: blockscout
+    name: Chiado explorer
```
Comment (paulbalaji) [inline chains/gnosischiadotestnet/metadata.yaml:5]:
```suggestion
    name: Chiado Explorer
```

File: chains/gnosischiadotestnet/metadata.yaml [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://gnosis-chiado.blockscout.com/api/v2
```
Comment (paulbalaji) [inline chains/gnosischiadotestnet/metadata.yaml:3]:
```suggestion
  - apiUrl: https://gnosis-chiado.blockscout.com/api
```

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**